shub/schedule.py

```diff
@@ -0,0 +1,41 @@
+from ConfigParser import NoSectionError, NoOptionError
+import click
+from click import ClickException
+from scrapinghub import Connection
+from shub import scrapycfg
```

Comment:
We could get something nice making this import lazy (moving it to inside `get_project_id_from_config()`): the command would not need Scrapy when you use `-p` argument.
If it doesn't complicate anything else, would be cool to have it. :)